src/rules.c

```diff
@@ -112,14 +115,14 @@ SchemaRule *SchemaRule_Create(SchemaRuleArgs *args, StrongRef ref, QueryError *s
     rule->lang_default = DEFAULT_LANGUAGE;
   }
 
-  rule->prefixes = array_new(sds, args->nprefixes);
+  rule->prefixes = array_new(HiddenUnicodeString*, args->nprefixes);
   for (int i = 0; i < args->nprefixes; ++i) {
-    sds p = sdsnew(args->prefixes[i]);
+    HiddenUnicodeString* p = NewHiddenUnicodeString(args->prefixes[i]);
```

Comment:
The need for them to be unicode comes from the redis keys being unicode since they need to be a prefix for a set of keys.
So I personally think that they can be unicode.